frontends/election-manager/src/components/test_deck_tally_report.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  ContestTally,
+  LogoMark,
+  Prose,
+  ReportSection,
+  TallyReportColumns,
+  TallyReportMetadata,
+  TallyReportTitle,
+} from '@votingworks/ui';
+import { Election, getPrecinctById, Tally } from '@votingworks/types';
+import { filterTalliesByParty } from '@votingworks/utils';
+
+const PrintableTallyReport = styled.div`
+  @page {
+    margin: 0;
+  }
+  section {
+    padding: 0.5in;
+    page-break-after: always;
+  }
```

Comment:
Just for my understanding: How did you land on these style additions? Put another way, where are these styles applied for the current test deck tally reports?